cmd/mayactl/app/command/commands.go

```diff
@@ -25,8 +25,8 @@ import (
 func NewMayaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "mayactl",
-		Short: "Maya means 'Magic'a tool for storage orchestration",
-		Long:  `Maya means 'Magic' a tool for storage orchestration`,
+		Short: "Mayactl means 'Magic' a tool for storage orchestration",
```

Comment:
mayactl doesn't mean magic :-) .. this should not be changed.

Reply:
Exactly , Its all 'Maya' :smile:  shouldn't be changed. changed to `Maya`